language/de-DE/de-DE.lib_joomla.ini

```
@@ -310,12 +309,16 @@ JLIB_FORM_BUTTON_SELECT="Auswählen"
 JLIB_FORM_CHANGE_IMAGE="Bild bearbeiten"
 JLIB_FORM_CHANGE_IMAGE_BUTTON="Bild-Button bearbeiten"
 JLIB_FORM_CHANGE_USER="Benutzer auswählen"
+JLIB_FORM_CONTAINS_INVALID_FIELDS="Das Formular kann nicht abgeschickt werden, da erforderlichen Daten fehlen.<br>Bitte die markierten Felder korigieren und erneut versuchen."
 JLIB_FORM_ERROR_FIELDS_CATEGORY_ERROR_EXTENSION_EMPTY="Das Erweiterungsattribut im Kategoriefeld ist leer."
 JLIB_FORM_ERROR_FIELDS_GROUPEDLIST_ELEMENT_NAME="Unbekannter Elementtyp: %s"
 JLIB_FORM_ERROR_NO_DATA="Keine Daten!"
 JLIB_FORM_ERROR_VALIDATE_FIELD="Ungültiges XML-Feld!"
 JLIB_FORM_ERROR_XML_FILE_DID_NOT_LOAD="XML-Datei nicht geladen!"
 JLIB_FORM_FIELD_INVALID="Ungültiges Feld:&#160;"
+JLIB_FORM_FIELD_INVALID_VALUE="Dieser Wert ist ungültig."
+JLIB_FORM_FIELD_REQUIRED_CHECK="Eine der Optionen muss ausgewählt sein."
+JLIB_FORM_FIELD_REQUIRED_VALUE="Der Feldwert darf nicht leer sein."
```

```suggestion
JLIB_FORM_FIELD_REQUIRED_VALUE="Der Wert darf nicht leer sein."
```

Haben wir im Install so drin.